.github/workflows/acceptance-tests.yml

```
@@ -467,4 +469,31 @@ jobs:
           PARALLEL_GO_TEST: 20
           CI: true
           TEST_REGEX: "^TestAccSTSAssumeRole"
-        run: make testacc
\ No newline at end of file
+        run: make testacc
+
+  search_index:
+    needs: [ change-detection ]
+    if: ${{ needs.change-detection.outputs.search_index == 'true' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' || github.event.label.name == 'run-testacc-cluster' || inputs.parent-event-name == 'release' }}
```

`github.event.label.name` allows you to run only this job if you add the specified label to the PR

```suggestion
    if: ${{ needs.change-detection.outputs.search_index == 'true' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' || github.event.label.name == 'run-testacc' || github.event.label.name == 'run-testacc-search-index' || inputs.parent-event-name == 'release' }}
```

---

oh, thanks, will change